app/services/parameter_safe_render.rb

```diff
@@ -0,0 +1,38 @@
+# Class to parse ERB with or without Safemode rendering. Needs a set
+# of variables, usually something like:
+#   @allowed_vars = { :host => @host }
+# so that <%= @host.name %> has the right @host variable
+#
+class ParameterSafeRender
+  include Foreman::Renderer
```

Comment:
including `UnattendedHelper` from `app/helpers/unattended_helper.rb` instead would also make `root_pass`, `grub_pass` and `ks_console` work

Reply:
Sounds good